src/core/Tw2Vector4Parameter.js

```diff
@@ -99,4 +99,14 @@ Tw2Vector4Parameter.prototype.SetIndexValue = function(index, value)
     }
 };
 
+Tw2Vector4Parameter.prototype.FillWith = function(number)
+{
+    if (/^-?[\d.]+(?:e-?\d+)?$/.test(number))
+    {
+        this.SetValue(number, number, number, number);
```

Comment:
isn't the argument to SetValue supposed to be an array?
and in general, wouldn't it be better to just add logic to SetValue - it the argument is a number then do "fill with" thing, otherwise just set the value?